spec/javascripts/global-bar-init.spec.js

```diff
@@ -10,7 +10,7 @@ describe('Global bar initialize', function () {
   })
 
   it('does not set cookie for blacklisted URL', function() {
```

Comment:
```suggestion
  it('does not set cookie for blocklisted URL', function() {
```

Can we replace it for the survey-related code too? 🙏

Reply:
Ah, Vanita already spotted it.

Reply:
Just spotted that - will do